### PR TITLE
qa_crowbarsetup: Setup Barbican service

### PR DESCRIPTION
**Pull request by toabctl**
When "want_barbican" is set, also deploy the barbican proposal.

### CODE REVIEW

**Comment by aplanas** [comment]
+1

**Comment by toabctl** [comment]
The barbican integration is not yet merged (see https://github.com/crowbar/crowbar-openstack/pull/392 ) but it's anyway useful to test  the mentioned PR.

**Comment by sayalilunkad** [comment]
+1

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -3081,6 +3082,9 @@ function deploy_single_proposal()
 
     # proposal filter
     case "$proposal" in
+        barbican)
+            [[ -n "$want_barbican" ]] || return
```
Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: would be nice to have a iscloudver 7plus check here...

Comment (toabctl) [inline scripts/qa_crowbarsetup.sh]: done

**Comment by dirkmueller** [comment]
+1

**Pull request merged**